extensions/2.0/Khronos/KHR_instancing/extensions/2.0/Khronos/KHR_instancing/README.md

```diff
@@ -0,0 +1,61 @@
+# KHR\_instancing
+
+## Khronos 3D Formats Working Group
+
+* John Cooke
+* TODO
+
+## Acknowledgments
+
+* TODO
+
+## Status
+
+Experimental
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension is specfically designed to enable GPU instancing which renders many copies of a single mesh at once using a small number of draw calls.  It's useful for things 
+like trees, grass, road signs, etc.  The TRANSFORM attribute allows the mesh to be displayed at many different locations with different rotations and scales.  The ID attribute 
+can be used to alter other parameters based on the run-time inplementation.
+
+## Extending Nodes with per instance attributes
+
+Instancing is defined by adding the `KHR_instancing` extension to any glTF node that has a mesh.  Instancing only applies to mesh nodes, there is no defined behavior for a node 
+with this extension that doesn't also have a mesh.  Applying to nodes rather than meshes allows the same mesh to be used by several nodes, instanced or otherwise.  The attributes 
+section contains accessor ids for the new TRANSFORM and ID attribute buffers.  For example, the following defines some instancing attributes to a node with mesh.  
+
+```json
+{
+    "nodes": [
+        {
+            "mesh": 0,
+            "name": "teapot",
+            "extensions": {
+                "KHR_instancing": {
+                    "attributes": {
+                        "TRANSFORM": 0,
```

Comment:
Thinking of instancing + animation. Allowed values for `animation.channel.target.path` are 
- translation
- rotation
- scale
- weights

https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#targetpath-white_check_mark

Do we also need `KHR_instancing` extension in `animation.channel` to specify TRANSFORM attribute as target.path?

```json
    "channels": [
        {
            ....,
            "KHR_instancing": {
                "sampler": 0,
                "target": {
                    "node": 0,
                    "path": "transform"
                }
            }
        }
    ]
```

Reply:
Thanks! Replied on https://github.com/KhronosGroup/glTF/pull/1691#issuecomment-546029405 for wider visibility.